src/components/RightPane/Map/UCIMap.tsx

```diff
@@ -296,7 +296,7 @@ export default class UCIMap extends PureComponent {
                             !event.start.toString().includes(DAYS[day]) ||
                             courses.has(event.sectionCode) || // Remove duplicate courses that appear in the calendar
                             !courses.add(event.sectionCode) ||
-                            FAKE_LOCATIONS.includes(event.bldg)
+                            FAKE_LOCATIONS.includes(event.bldg.trim())
```

Comment:
Good patch!

Do you know how this occurs? Ideally, we'd be handling the data in such a way that doesn't require us to patch it like this

Reply:
I will inspect this! Thanks!

Reply:
Ok i'm sure the bldg field returned by PeterPortal api actually has a trailing whitespace when it's "TBA." So do you think this is a proper place to strip(trim) the bldg? @bevm0

Reply:
Dang, what the heck, the "TBA" has embedded whitespaces in the HTML itself from the SOC, i.e. the `&nbsp;`
![image](https://user-images.githubusercontent.com/66761366/222855356-14acb27d-d137-4637-84ed-bd3707d4ff9e.png)

Reply:
uhh I just saw your comment right after Eric merged it ; ( 

So the note hasn't been added yet...

Reply:
Oops I didn't realize Brian wanted more changes on this 😐. 
Just read the comment and that change would be appreciated so just open another PR for it